Dockerfile

```
@@ -0,0 +1,18 @@
+FROM node:16.16.0-alpine AS builder
+
+WORKDIR /app
+COPY package*.json ./
+RUN npm ci
+COPY tsconfig*.json ./
+COPY src src
+RUN npm run build
```

You can simplify this.

```suggestion
WORKDIR /app
COPY . .
RUN npm ci \
    npm run build
```


 This will lead you to use fewer build steps and, as a result, reduce the number of cached layers

---

I don't think it needs to be simplified, because it will have a negative impact on build speed. Reducing the number of cached layers may be useful if we have really big containers, this one is just `136.66 MB`. I specifically designed it to take advantage of layer caching to reduce build time. It is so for a reason.